api/api/specs/routers.yaml

```diff
@@ -796,6 +796,7 @@ components:
       nullable: true
       required:
```

Comment:
If only one of the 2 props is supposed to be set, should we remove both from `required`?

Reply:
Hmm that's true, I was thinking of keeping both `required` and letting requests to the API use `null` values for an array/string (`[]` and `""`) to denote the empty field but I guess the reverse works too :)